### PR TITLE
[Force Update] V1

### PR DESCRIPTION
**Pull request by gogolon**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Miscellaneous
+.DS_Store
```
Comment (PiotrRogulski) [inline .gitignore:2]: Untrack all .DS_Stores

File: packages/force_update/backend/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿namespace backend;
```
Comment (PiotrRogulski) [inline packages/force_update/backend/Class1.cs:1]: Are those files supposed to be here? I think with the current configuration, the backend will be included as part of the Flutter package

Comment (gogolon) [reply packages/force_update/backend/Class1.cs:1]: good question. I think it should be in repo, but definitely should not be in package. I'll think whether to remove it completely or just move the backend folder higher to make sure it's not included in the package

Comment (gogolon) [reply packages/force_update/backend/Class1.cs:1]: moved backend folder up

File: packages/.gitignore
Comment (RobertOdrowaz) [inline packages/.gitignore:1]: I don't think it's necessary since you already have it in root

File: packages/force_update/mobile/analysis_options.yaml
Comment (RobertOdrowaz) [inline packages/force_update/mobile/analysis_options.yaml:1]: I think we should use leancode_lint here. Also add a new line at the end

File: packages/force_update/mobile/example/.gitignore [hunk 3]
```diff
@@ -3,7 +3,7 @@
 *.log
 *.pyc
 *.swp
-.DS_Store
+**.DS_Store
```
Comment (RobertOdrowaz) [inline packages/force_update/mobile/example/.gitignore:6]: I don't think you need to change this

File: packages/force_update/mobile/lib/src/app_version.dart [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  bool operator >(T other) => compareTo(other) > 0;
+}
+
+class AppVersion implements Comparable<AppVersion> {
+  const AppVersion({required this.version});
+
+  final String version;
+
+  @override
+  int compareTo(other) {
```
Comment (RobertOdrowaz) [inline packages/force_update/mobile/lib/src/app_version.dart:14]: Using this package https://pub.dev/packages/version seems safer than writing this logic from scratch. I believe there are quite a lot of edge-cases here

**Review by RobertOdrowaz — APPROVED** [review]

File: packages/force_update/mobile/lib/src/app_version.dart (hunk 4 above)
Comment (gogolon) [reply packages/force_update/mobile/lib/src/app_version.dart]: replaced with the package

**Pull request merged**